Gemfile

```diff
@@ -3,11 +3,12 @@ git_source(:github) { |repo_name| "https://github.com/#{repo_name}.git" }
 
 ruby '~> 3.2'
 
-gem 'rails', '~> 7.0.8'
+gem 'rails', '~> 7.1.0'
 
 gem 'activerecord-import', '>= 1.0.2'
 gem 'aws-sdk-s3'
 gem 'bloomfilter-rb'
+gem 'redis'
```

Comment:
this upgrade also upgraded the bloomfilter-rb gem, which caused a couple of tests to fail. when  i looked into it, it appears the bloomfield gem used to install production redis, but [5 years ago made it dev-only.](https://github.com/igrigorik/bloomfilter-rb/pull/45), so I added `redis` to our Gemfile. this might not be the best way to solve it, so open to suggestions! 

(i do think i'd prefer to stay on this more recent version of bloomfilter as the old one was over a decade old.)

Reply:
I think that's fine for the sake of this PR

However,  I remember checking a few years ago and bloom filter was never enabled. I checked today and it is still disabled. I think given that this appears to be unused code I would consider making a separate PR and just removing it entirely

Reply:
oh! okay, i can do that in a followup PR.